.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,20 +3,19 @@
 | Branch?       | 4.4 for features / 3.4 or 4.3 for bug fixes <!-- see below -->
 | Bug fix?      | yes/no
 | New feature?  | yes/no <!-- please update src/**/CHANGELOG.md files -->
-| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- please update UPGRADE-*.md and src/**/CHANGELOG.md files -->
-| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
-| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
+| Tickets       | Fix #... <!-- prefix each issue number with "Fix #", if any -->
 | License       | MIT
 | Doc PR        | symfony/symfony-docs#... <!-- required for new features -->
-
 <!--
 Replace this notice by a short README for your feature/bugfix. This will help people
 understand your PR and can be used as a start for the documentation.
 
 Additionally (see https://symfony.com/roadmap):
+ - Always add tests and ensure they pass.
+ - Never break backward compatibility (see https://symfony.com/bc).
  - Bug fixes must be submitted against the lowest maintained branch where they apply
-   (lowest branches are regularly merged to upper ones so they get the fixes too).
+   (lowest branches are regularly merged to upper ones so they get the fixes too.)
```

Comment:
doesn't look right to put the `.` before the `)`... but maybe it's me.

Reply:
it's not you, it's English :)